lib/shared/addon/components/cluster-driver/driver-gke/component.js

```diff
@@ -954,6 +962,14 @@ export default Component.extend(ClusterDriver, {
       }
     }
 
+    if (get(config, 'ipAllocationPolicy.clusterSecondaryRangeName') && get(config, 'ipAllocationPolicy.clusterIpv4CidrBlock')) {
+      set(config, 'ipAllocationPolicy.clusterIpv4CidrBlock', null);
+    }
+
+    if (get(config, 'ipAllocationPolicy.servicesSecondaryRangeName') && get(config, 'ipAllocationPolicy.servicesIpv4CidrBlock')) {
+      set(config, 'ipAllocationPolicy.servicesIpv4CidrBlock', null);
+    }
+
```

Comment:
My networking knowledge is really poor so this might be a reasonable path but it seems odd to me. 

I would think we'd want the UX to either make these fields mutually exclusive or to inform the user that there was a collision via an error message so they could remedy it as they see fit.

Feel free to ignore if my assessment is wrong.

Reply:
In the UI, if you select a range, we disable the manual input option and replace it with the IP of the range selected.

I think that's okay - a better UX would probably be radio buttons to explicitly chose, but given Ember is legacy, this is the minimum to fix the isue.